src/ref.rs

```diff
@@ -570,278 +570,6 @@ where
     }
 }
 
-impl<B, T> Ref<B, T>
-where
-    B: ByteSlice,
-    T: Unaligned + KnownLayout + Immutable + ?Sized,
-{
-    /// Constructs a `Ref` for a type with no alignment requirement from a byte
-    /// slice.
-    ///
-    /// If the length of `source` is not a [valid size of `T`][valid-size], this
-    /// returns `Err`.
-    ///
-    /// `T` may be a sized type, a slice, or a [slice DST][slice-dst].
-    ///
-    /// [valid-size]: crate#what-is-a-valid-size
-    /// [slice-dst]: KnownLayout#dynamically-sized-types
-    ///
-    /// # Compile-Time Assertions
-    ///
-    /// This method cannot yet be used on unsized types whose dynamically-sized
-    /// component is zero-sized. Attempting to use this method on such types
-    /// results in a compile-time assertion error; e.g.:
-    ///
-    /// ```compile_fail,E0080
-    /// use zerocopy::*;
-    /// # use zerocopy_derive::*;
-    ///
-    /// #[derive(Immutable, KnownLayout, Unaligned)]
-    /// #[repr(C, packed)]
-    /// struct ZSTy {
-    ///     leading_sized: u16,
-    ///     trailing_dst: [()],
-    /// }
-    ///
-    /// let f = Ref::<&[u8], ZSTy>::unaligned_from_bytes(&b"UU"[..]); // ⚠ Compile Error!
-    /// ```
-    #[must_use = "has no side effects"]
-    #[inline(always)]
-    pub fn unaligned_from_bytes(source: B) -> Result<Ref<B, T>, SizeError<B, T>> {
```

Comment:
Should these be removals, or deprecations?

Reply:
Removals; they're [new as of 0.8](https://docs.rs/zerocopy/0.7.35/zerocopy/struct.Ref.html). We've already deprecated all of the 0.7 methods, which have a different naming scheme.